3/Kox_sum/kox_sum_simple_task.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <vector>
+#include <ctime>
+#include <fstream>
+
+void print_nxm_mat_double(std::vector<std::vector<double>> Mat)
+{
+    for(std::vector<double> el : Mat)
+    {
+        for(double num : el)
+        {
+            std::cout << num << " ";
+        }
+        std::cout << std::endl;
+    }
+}
+
+
+int main()
```

Comment:
We have to discuss what this program does and what it has to be doing. It least as far as I understand you've misinterpreted my explanation.